src/components/Banners/ErrorMessage.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+import { isKonnectorTwoFAError } from 'lib/konnectors'
+import Button from 'cozy-ui/react/Button'
+import PropTypes from 'prop-types'
+
+import Banner from 'components/Banners/Banner'
+import { getMostAccurateErrorKey, isKonnectorKnownError } from 'lib/konnectors'
+
+export class ErrorMessage extends Component {
+  constructor(props) {
+    super(props)
+    this.getButtonComponent = this.getButtonComponent.bind(this)
+    this.getErrorTitle = this.getErrorTitle.bind(this)
+    this.getErrorDescription = this.getErrorDescription.bind(this)
+  }
+
+  getButtonComponent(error) {
```

Comment:
Nit: I like to have internal methods that return components to have `render` in them : `renderButtonComponent`.

Nit: I find Component a bit superfluous: `renderButton`